examples/pytorch/graphsim/README.md

```diff
@@ -0,0 +1,72 @@
+# GraphParticleSim
+## DGL Implementation of Interaction-Network paper.
+
+This DGL example implements the GNN model proposed in the paper [Interaction Network](https://arxiv.org/abs/1612.00222.pdf). 
+
+GraphParticleSim implementor
+----------------------
+This example was implemented by [Ericcsr](https://github.com/Ericcsr) during his Internship work at the AWS Shanghai AI Lab.
+
+The graph dataset used in this example 
+---------------------------------------
+This Example uses Datasets Generate By Physics N-Body Simulator adapted from [This Repo](https://github.com/jsikyoon/Interaction-networks_tensorflow)
+
+n_body:
+    - n Particles/Nodes
+    - Complete Bidirectional Graph
+    - 10 trajectories should be generated
+    - 1000 steps of simulation per trajectory
+
+Dependency
+--------------------------------
+```
+sudo apt install ffmpeg
+pip install opencv-python
+```
```

Comment:
Only list the dependency package and version, like:
Dependencies
numpy 1.19.4
pytorch 1.7.1
dgl 0.5.3
networkx 2.5
matplotlib 3.3.4